packages/ui-bindings/EconomicEvent/CreateEconomicEvent.tsx

```diff
@@ -0,0 +1,67 @@
+import { connect } from "react-redux";
```

Comment:
Please credit yourself for this beautiful file!

Reply:
Added header comments